.github/workflows/formatter.yml

```diff
@@ -0,0 +1,37 @@
+name: Format
+
+on:
+  pull_request:
+    paths: ['**.cs']
+  push:
```

Comment:
We definitely don't want to run this on pushes to `master` branch as the `master` branch is protected (it will not be able to commit changes to protected branch). I think it should run only on PRs.